.travis.yml

```diff
@@ -77,6 +77,20 @@ jobs:
         - make image-cephcsi || travis_terminate 1;
         - scripts/travis-functest.sh v1.17.0 || travis_terminate 1;
 
+    - name: cephcsi with kube 1.15.6 with psp support
```

Comment:
this need to be tested if there  is a code change or helm charts change not for static template change